samples/bluetooth/central_and_pripheral_hr/README.rst

```diff
@@ -0,0 +1,170 @@
+
+1. |connect_terminal|
+#. Program the other kit with :ref:`zephyr:peripheral_hr` samples and reset it.
+#. Wait until the HRS is detected by the central.
+   Observe that **LED 2** is on.
+#. In the terminal window, check for information similar to the following:
+        Heart Rate Sensor body location: Chest
+        Heart Rate Measurement notification received:
+
+            Heart Rate Measurement Value Format: 8 - bit
+            Sensor Contact detected: 1
+            Sensor Contact supported: 1
+            Energy Expended present: 0
+            RR-Intervals present: 0
+
+            Heart Rate Measurement Value: 134 bpm
+
+   Notifications will be displayed periodically with a frequency determined by the HR server.
+
+#. Program another kit with :ref:`zephyr:bluetooth_central_hr` and reset it.
+#. Wait until central is connected to your development kit.
+   Observe that **LED 3** is on.
+#. In terminal windows connected to :ref:`zephyr:bluetooth_central_hr`, check for information similar to following:
+
+        [NOTIFICATION] data 0x20006779 length 2
+
+#. Sample is working as relay now.
+
+Testing with nRF Connect for Desktop
+------------------------------------
+1. |connect terminal|
+#. Reset the kit.
+#. Start `nRF Connect for Desktop`_ and select the connected dongle that is used for communication.
+#. Go to the :guilabel:`Server setup` tab.
+   Click the dongle configuration and select :guilabel:`Load setup`.
+   Load the :file:`hr_service.ncs` file that is located under :file:`samples/bluetooth/central_and_peripheral_hr` in the |NCS| folder structure.
+#. Click :guilabel:`Apply to device`.
+#. Go to the :guilabel:`Connection Map` tab.
+   Click the dongle configuration and select :guilabel:`Advertising setup`.
+
+   The current version of nRF Connect can store the advertising setup.
+
+   #. Click :guilabel:`Load setup`.
+      Load the :file:`hrs_adv_setup.ncs` file that is located under :file:`samples/bluetooth/central_and_peripheral_hr` in the |NCS| folder structure.
+   #. Click :guilabel:`Apply` and :guilabel:`Close`.
+
+#. In the Adapter settings, choose :guilabel:`Start advertising`.
+#. Wait until the kit that runs the Central and Peripheral HRS connects.
+    Observe that **LED 2** is on.
+#. Explore the :guilabel:`Heart Rate Measurement` characteristic.
+   Write value `06 80` and click :guilabel:`Play` button to send notification.
+   In the terminal window, check for information similar to the following:
+
+        Heart Rate Sensor body location: Chest
+        Heart Rate Measurement notification received:
+
+            Heart Rate Measurement Value Format: 8 - bit
+            Sensor Contact detected: 1
+            Sensor Contact supported: 1
+            Energy Expended present: 0
+            RR-Intervals present: 0
+
+            Heart Rate Measurement Value: 128 bpm
+
+#. The `nRF Connect for Desktop`_ also detects the Central and Peripheral HRS sample Heart Rate Service.
```

Comment:
The Bluetooth Low Energy app? I believe this is not a step anymore, but the two below  (Enable and Write) could be steps.

Reply:
Yes, but we use everywhere just `nRF Connect for Desktop`. Should I change it ?

Reply:
I would say yes. Maybe in the first occurrence, we could explain that this means the Bluetooth Low Energy app found in the nRF Connect for Desktop.

Reply:
Ok at begin of the Testing section I changed it to the nRF Connect for Desktop Bluetooth Low Energy application